requirements_dev.txt

```diff
@@ -1,6 +1,5 @@
 invoke~=2.2
 otelib>=0.3,<0.4.0
-pre-commit~=3.6
```

Comment:
Should we not still have pre-commit as requirement for development work? Pre-commit should run for each commit...

Reply:
The docs requirements are an implicit part of the dev requirements. Explicit in the `setup.py` file.

See these lines: https://github.com/EMMC-ASBL/oteapi-dlite/blob/master/setup.py#L63-L69

Reply:
OK, this requires that people install dev requirements by pip install .[dev] and not -r requirements_dev.txt.
This is fine for me.